app/javascript/src/utils/index.js

```diff
@@ -1,11 +1,26 @@
 import dayjs from "dayjs";
 import * as R from "ramda";
 import slugify from "slugify";
+import { v4 as getNewId } from "uuid";
```

Comment:
If you needed to name this as `getNewKey`, then why did you import it under the `geNewId` alias?

Reply:
That was intentional. I wanted to abstract key generation. But since `getNewId` and `getNewKey` both used UUID in the background, I created an alias for `getNewKey`. Both `getNewId` and `getNewKey` are in use. The reason I chose to do that was, if I change my key generation algorithm, I didn't want to modify by a lot. Now if I want to change my keygen algorithm, I just have to replace the RHS of `getNewKey`.😁

Reply:
@vivek-viswam-rv https://www.loom.com/share/7bdd312e575d401db3dba99c4b35e5ea

Reply:
Thanks @yedhink pointing it out, I'll modify it accordingly. Also is it ok if I push these changes to the PR that solves #7?